aerleon/lib/aclgenerator.py

```diff
@@ -294,53 +294,57 @@ def __init__(self, pol, exp_info):
         """Initialise an ACLGenerator.  Store policy structure for processing."""
         supported_tokens, supported_sub_tokens = self._GetSupportedTokens()
 
+        pol.filters[:] = [
+            (header, terms)
+            for (header, terms) in pol.filters
+            if (self._PLATFORM in (header, terms)[0].platforms)
```

Comment:
Can this be simplified?
```suggestion
            if (self._PLATFORM in header.platforms)
```